### PR TITLE
Don't expect storage buffers to work in vertex stage

### LINKED ISSUE #113: createPipelineLayout tests expect storage buffers to work in vertex stage
I think I made a mistake when generalizing these tests to cover more cases.

- `cts:validation/createPipelineLayout:dynamic_offsets_are_only_allowed_on_buffers~`
- `cts:validation/createPipelineLayout:number_of_bind_group_layouts_exceeds_the_maximum_value~`

### PR DESCRIPTION
**Pull request by kainino0x**
Fixes #113

### CODE REVIEW

File: src/suites/cts/validation/createPipelineLayout.spec.ts [hunk 1]
```diff
@@ -61,39 +61,38 @@ g.test('number of dynamic buffers exceeds the maximum value', async t => {
...
   )
 );
 
 g.test('number of bind group layouts exceeds the maximum value', async t => {
-  const { visibility, type } = t.params;
+  const type: GPUBindingType = t.params.type;
 
   const bindGroupLayoutDescriptor: GPUBindGroupLayoutDescriptor = {
-    bindings: [{ binding: 0, visibility, type }],
+    bindings: [{ binding: 0, visibility: C.ShaderStage.Fragment, type }],
```
Comment (kvark) [inline src/suites/cts/validation/createPipelineLayout.spec.ts]: might add a comment that we'll still need to test the read-only storage in VS stage once it's in

Comment (kvark) [inline src/suites/cts/validation/createPipelineLayout.spec.ts]: also, why discriminate against compute stage? :)

Comment (austinEng) [reply src/suites/cts/validation/createPipelineLayout.spec.ts]: It seems that most of the binding count limits with all permutations of usage and visibility happens in createBindGroupLayout.spec.ts `max resources per stage/in pipeline layout` and `max resources per stage/in bind group layout`.

So if the purpose of the test is `number of bind group layouts exceeds the maximum value`, then I question whether we need any `bindings:` and can instead just make a pipelineLayout with 4, and then 4 + 1 empty bind group layouts.

**Comment by kainino0x** [comment]
TODO: `kBindingTypeInfo['readonly-storage-buffer'].validStages` should allow vertex shader

File: src/suites/cts/validation/createPipelineLayout.spec.ts (hunk 1 above)
Comment (kainino0x) [reply src/suites/cts/validation/createPipelineLayout.spec.ts]: yes, and in fact I guess it doesn't really make sense to test multiple binding types here. Maybe the best thing is to test multiple values for `visibility` and just one for the binding type.

Comment (kainino0x) [reply src/suites/cts/validation/createPipelineLayout.spec.ts]: Also, IIRC the change to this test is unrelated to the title of the PR, sorry (rushed this PR out at end of day)

**Comment by kainino0x** [comment]
PTAL

**Pull request merged**